.env.example

```diff
@@ -20,7 +20,6 @@ DB_PASSWORD=secret
 
 # These are the recommended default values
 BROADCAST_DRIVER=log
-CACHE_DRIVER=file
```

Comment:
Was ist denn stattdessen? `database`? Im Docker-Setup wird so oder so ein Redis hochgefahren, nehmen wir dann das? Evtl. in `docker/.env.app` anpassen dann. (Nutzen wir Cache eigentlich schon irgendwo anders? Oder ist das dann das erste Mal?)

Reply:
Wir nutzen kein Redis. Bislang wurde generell noch kein Caching genutzt. Die ENV Variable ist daher bisher ungenutzt gewesen. Damit der Default aus dem Code greift hab ich es auch aus der .env.example entfernt.